strr-api/src/strr_api/resources/application.py

```diff
@@ -111,6 +112,9 @@ def create_application():
         if not valid:
             return error_response(message="Invalid request", http_status=HTTPStatus.BAD_REQUEST, errors=errors)
         validate_request(json_input)
+        roles = AccountService.list_account_roles(account_id=account_id)
```

Comment:
How do you plan on handling platform related roles in future? Based on application type?